ui/src/tool-panel/location-track/location-track-infobox-duplicate-of.tsx

```diff
@@ -52,19 +53,85 @@ const LocationTrackDuplicateInfoIcon: React.FC<{
     );
 };
 
-const LocationTrackDuplicateTrackNumberWarning: React.FC<
-    LocationTrackDuplicateTrackNumberWarningProps
-> = ({ differingTrackNumberName }) => {
-    const { t } = useTranslation();
-    return (
-        <LocationTrackDuplicateInfoIcon
-            msg={t('tool-panel.location-track.duplicate-on-different-track-number', {
-                trackNumber: differingTrackNumberName,
-            })}
-            type={'ERROR'}
-        />
-    );
-};
+function checkAndNotifyImplicitDuplicate(
```

Comment:
Kukin eri huomiotyyppi funktioitu omaan check-and-notify-funktioonsa. En ole kauhean tyytyväinen näiden nimeämisiin vaikka niitä sai miettiäkin pitkään, ja se olisi haisu siitä että näissä on abstraktion tasolla jotain pielessä. Näissä kaikissa on sama pohjarakenne: Tarkista predikaatti ja jos se on totta, palauta huomio-olio jonka parametrisaatio muuttuu per error, muuten palauta `undefined`. En oikein nää että näitä voisi muuttaa mihinkään muuhun kauhean järkevästi ilman, että predikaatin tarkastelu ja/tai huomio-olion luonti ja parametrisaatio siirtyisi takaisin kutsupaikkaan, johon niitä ei oikeasti haluttaisi

Reply:
Sisältönä ihan asialliset. Nimet on joo vähän pitkiä plus "notify" kielii että funkkari heittäisi jonkun popupin tms ja sitähän ne ei tee. Ehkä joku getNotificationForX olisi siinä mielessä parempi, mutta ei yhtään lyhyempi. Superpitkät camelCaset on aina ärsyttävän kankeita lukea.

Ehkä ...LocationTrackOnDifferentTrackNumber voisi olla vain ...WrongTrackNumber koska raiteitahan tässä tietenkin käsitellään (eikä LocationTrack-sanaa noissa kaikissa muissakaan ole mukana). Tuo OverlappingDuplicateOfDifferentLocationTrack on vähän hankalampi. Ehkä sen voisi ajatella myös niin että siinä on kyseessä ConflictingDuplicateStatus tms.